src/styles/variables.scss

```diff
@@ -46,6 +46,16 @@ $grey-7: #202020;
 $brand: #bd1c2b;
 $highlight: #58cef4;
 
+// Tint and shade functions from
+// https://css-tricks.com/snippets/sass/tint-shade-functions
+@function tint($color, $percent) {
+  @return mix(white, $color, $percent);
+}
+
+@function shade($color, $percent) {
+  @return mix(black, $color, $percent);
+}
```

Comment:
I think ideally these utilities should live in a separate module which is included before variables. Working is better than non-working and we don't have such a module yet, so I'd keep this as-is in this PR.

Reply:
sounds good